app.py

```diff
@@ -289,21 +289,38 @@ def month_to_string(month):
 # Make a call to Nookipedia's Cargo API using supplied parameters:
 @cache.memoize(3600)
 def call_cargo(parameters, request_args): # Request args are passed in just for the sake of caching
+    cargoquery = []
     try:
-        r = requests.get(url = BASE_URL_API, params = parameters)
+        # The default query size is 50 normally, we can actually change it here if we wanted
+        cargolimit = int(parameters.get('limit','50'))
+        # Copy the current parameters, we'll be changing them a bit in the loop but we want the original still
+        nestedparameters = parameters.copy()
+        # Set up the offset, if our cargolimit is more than cargomax we'll end up doing more than one query with an offset
+        while True:
+            nestedparameters['limit'] = str(cargolimit-len(cargoquery)) # Get cargomax items or less at a time
+            if nestedparameters['limit']=='0': #Check if we've hit the limit
+                break
+            nestedparameters['offset'] = str(len(cargoquery))
+            r = requests.get(url = BASE_URL_API, params = nestedparameters)
+            cargochunk = r.json()['cargoquery']
+            if len(cargochunk) == 0: #Check if we've hit the end
+                break
+            cargoquery.extend(cargochunk)
         print('Return: {}'.format(str(r)))
     except:
         print('Return: {}'.format(str(r)))
+        import traceback
```

Comment:
Can we move the import to the top of the file alongside the rest?

Reply:
+ update `requirements.txt` with traceback

Reply:
I completely forgot I put that there, it was just to print the stack trace, it can actually be removed
traceback is part of the standard library, so technically we wouldn't need to add it, but it'd be better to drop that from the file entirely, it's not really required except for debugging

Reply:
Gotcha! Thanks.